grease-cli/tests/llvm-bc/declare-in-override.c

```diff
@@ -1,5 +1,7 @@
 /* Copyright (c) Galois, Inc. 2024 */
 
+// CFLAGS: -emit-llvm -frecord-command-line
```

Comment:
To avoid duplicating these flags across all `llvm-bc` tests, I wonder if we should make some pre-defined flag groups available (e.g., something like `// CFLAGS: $LLVM_BC_FLAGS`).

Reply:
Good idea, ported the script to Python and added this feature.